stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1897,3 +1897,46 @@ end
 
 normalize(x) = x / norm(x)
 normalize(x, p::Real) = x / norm(x, p)
+
+"""
+    copytotri!(uplo, A, B) -> B
+
+Copies a triangular part of a matrix `A` to another matrix `B`.
+`uplo` specifies the part of the matrix `A` to be copied to `B`.
+Set `uplo = 'L'` for the lower triangular part or `uplo = 'U'
+for the upper triangular part.
+
+!!! compat "Julia 1.11"
+    `copytotri!` requires at least Julia 1.11.
+
+# Examples
+```jldoctest
+julia> A = [1 2 ; 3 4];
+
+julia> B = [0 0 ; 0 0];
+
+julia> copytotri!('L', A, B)
+2×2 Matrix{Int64}:
+ 1  0
+ 3  4
+```
+"""
+function copytotri!(uplo::AbstractChar, A::AbstractMatrix, B::AbstractMatrix)
+    require_one_based_indexing(A, B)
+    LinearAlgebra.BLAS.chkuplo(uplo)
+    m, n = size(A)
+    if uplo == 'U'
+        @inbounds for j=1:n
+            @inbounds for i=1:min(j,m)
```

Comment:
The upper `@inbounds` applies to the whole block, so this one should not be necessary:

```suggestion
            for i=1:min(j,m)
```

Same below.

Reply:
Perhaps the `@inbounds` may be applied directly to the actual indexing operation on line 1931, and similar?

Reply:
Good idea!